runelite-client/src/main/java/net/runelite/client/plugins/friendnotes/FriendNotesPlugin.java

```diff
@@ -139,8 +154,10 @@ public void onMenuEntryAdded(MenuEntryAdded event)
 		// look for "Message" on friends list
 		if (groupId == WidgetInfo.FRIENDS_LIST.getGroupId() && event.getOption().equals("Message"))
 		{
-			setCurrentFriend(event.getTarget());
+			final String sanitizedTarget = Text.removeTags(event.getTarget());
```

Comment:
The Text.removeTags is mostly not necessary here, the friend list is not colored.

Reply:
Okay. Do you think it's safe to skip sanitizing the menuTarget in [onMenuOptionClicked()](https://github.com/runelite/runelite/pull/2714/files/11debd38181441f9851ea6310f5e2b9b2c37a11f#diff-39284c20a78808105c9beb0137d2adb0R188) as well?

Reply:
Nope, that one has color codes in it 😜

Reply:
Done.